colossalai/booster/plugin/hybrid_parallel_plugin.py

```diff
@@ -208,14 +208,15 @@ def __init__(
             cpu_offload: bool = False,    # cpu offload
             dp_process_group: Optional[ProcessGroup] = None,    # the dp pg for comm
             tp_process_group: Optional[ProcessGroup] = None,    # if using tp
-            forced_dtype: Optional[torch.dtype] = None):
+            forced_dtype: Optional[torch.dtype] = None,
+            extra_dp_process_group: Optional[ProcessGroup] = None):
```

Comment:
If this is only used with MOE, how about moving this to moe hybriad parallel plugin?

Reply:
done